.github/workflows/validate-pr.yml

```diff
@@ -0,0 +1,38 @@
+name: validate-pr
+
+on:
+  pull_request:
+    types: [opened, edited, labeled, unlabeled]
+
+jobs:
+  check-changelog:
+    if: contains(github.event.pull_request.labels.*.name, 'impact/changelog')
```

Comment:
Probably a good start, but we should also consider _any_ `impact/` label to be generally "changelog worthy". Perhaps it should check if any of those are applied that it _also_ has a `impact/changelog` https://github.com/moby/moby/labels?q=impact

Thinking of `impact/deprecation`, `impact/api` at least.


Also considering it a release-note should always have an `area/` label attached, so that we can group the release notes in the right section(s).

Reply:
Changed to any `impact/` label and also added an `area/` requirement.